docs/1-the-manual-menace/2-argocd.md

```diff
@@ -76,15 +76,16 @@ When something is seen as not matching the required state in Git, an application
     namespaces:
       - ${TEAM_NAME}-ci-cd
     argocd_cr:
-      repositoryCredentials: |
-        - url: https://${GIT_SERVER}
+      initialRepositories: | 
```

Comment:
putting my @pabrahamsson hat on, I believe he'll want to see this whitespace char gone 😉  

But yeah Im happy to merge if no one has any other qualms!

![bitmoji](https://sdk.bitmoji.com/render/panel/618bbad8-01ad-49c4-b1ea-2d84834a1f1a-5861b8d6-6071-4533-939a-16117539af15-v1.png?transparent=1&palette=1&width=246)

Reply:
An I know im being an absolute pedant with this change req 🍨

Reply:
That's cool ;) I've removed the extra whitespace